cpp/dolfin/mesh/Mesh.cpp

```diff
@@ -23,6 +23,7 @@
 
 using namespace dolfin;
 using namespace dolfin::mesh;
+using namespace dolfin::io;
```

Comment:
Remove this. Don't use 'inside' other namespaces (in the case the `mesh` namespace).